app/presenters/tree_builder_ops_rbac.rb

```diff
@@ -58,6 +58,6 @@ def x_get_tree_custom_kids(object_hash, count_only, _options)
   end
 
   def x_get_tree_tenant_kids(object, count_only)
-    count_only_or_objects(count_only, object.children, "name")
+    count_only_or_objects(count_only, object.children.in_my_region, "name")
```

Comment:
this is not needed, because object is already in one region and his children will be also the same region. I can show it you in my env.

Reply:
So there's no way for a `Tenant` to have a child that's in another region. Noted :)